### ISSUE TITLE
Sound clicks in Vexx

### ISSUE THREAD

**Issue by HelicopterP** [PCSX2/pcsx2#3846]
***PCSX2 version:***
v1.7.0-dev-465-g2608673ca

***PCSX2 options:***
default audio settings, but changing them doesn't make any difference

***Plugins used:***
SPU2 built into v1.7.0-dev-465-g2608673ca

***Description of the issue:***
Audio pops and clicks

***How to reproduce the issue:***
Start the game and listen. The issue is more prominent during FMV videos, but you can also hear clicks during gameplay.

***Last known version to work:***
`As far as I'm aware this game never had propper sound

***PC specifications:***
I7 8700, MSI Z370-A PRO, integrated motherboard audio

Blockdump with a couple of seconds of intro video
[Vexx (2021-00-06 19-04-38).zip](https://github.com/PCSX2/pcsx2/files/5776829/Vexx.2021-00-06.19-04-38.zip)

**Comment by MrCK1** [comment]
@HelicopterP Did you check with ASync Mix On/Off to try and resolve the issue?

**Comment by HelicopterP** [comment]
Unfortunately turning async mix on or off doesn't make any difference

**Comment by MrCK1** [comment]
@HelicopterP Can you try the build here and see if if helps with the sound? Thanks. 
https://github.com/PCSX2/pcsx2/suites/1577068086/artifacts/28498350

**Comment by HelicopterP** [comment]
Unfortunatelly it didn't change the situation at all.

**Comment by HelicopterP** [comment]
Recent work on SPU2 and IOP did not improve this game.

**Comment by HelicopterP** [comment]
I just realized, that this bug is very simple to reproduce with a blockdump, so I updated my initial report with a small dump with a begginnig of intro video. Maybe sound clicks during gameplay is another bug entirely, but at least this is something.

With all the work being done on IPU and SPU2 maybe @refractionpcsx2 could check this out :P

**Comment by refractionpcsx2** [comment]
I feel picked on... :P

**Comment by refractionpcsx2** [comment]
This should be fixed when #4171 gets merged, game is kinda dumb :P

**Comment by HelicopterP** [comment]
Yep, videos are most definitely fixed. There are some clicks during gameplay, but A LOT less noticeable than before. I'm not sure that those aren't supposed to be there, tbh. Whne I played this game on a real PS2 I did not hear anything abnormal, but that could be due to me daisy chaining a lot of analog devices and raised noise floor as a result of this. I am too lazy to dig up a PS2 again, connect it to OSSC, etc... I found a gameplay video from this game running on Gamecube and there are sound clicks during gameplay, but in the right channel. I suppose, developers didn't really care about sound - it was OK playing through crappy TV speakers, so who cares about headphones...

TL DR It's fixed as far as I can tell.

**Comment by refractionpcsx2** [comment]
cool :) I've just got some other bugs to fix before I submit this...

**Comment by HelicopterP** [comment]
The latest master behaves the same way as a early version of that pull-request did. I played this game a bit more and a little bit of clickiness is only found during tutorial stage. Everything is absolutely fine, when the game starts propper. Could be a game bug really...
Curiously, the same kind of blips/clicks can be heard during the intro for Sphinx and the Cursed Mummy (after starting the new game). Or maybe this two are completly unrelated. I don't know. Anyway, this is getting offtopic.

**Comment by refractionpcsx2** [comment]
I got one of our testers to check Sphinx as he has the game and he said there wasn't really any clicks/beeps that he noticed, maybe one or two caused by stutters, but that's about it.

**Comment by HelicopterP** [comment]
There aren't really any clicks during normal gameplay, but there are (and aways were AFAIK) during this scene right in the beginning (shown on screenshot). It's not caused by stutters or low framerate. I guess, some people just aren't sensitive to this. Cliks are also quite, which may explain whysome people don't notice the issue. I'll try to make a block dump and open a new issue for Sphinx.
![pcsx2 2021-01-31 17-50-14](https://user-images.githubusercontent.com/4709459/106388032-af0a0300-63ed-11eb-9da8-70b6b3ec7481.jpg)

**Comment by refractionpcsx2** [comment]
Alright, thanks :)
